common/util/FontUtils.cpp

```diff
@@ -320,6 +348,50 @@ std::string GameTextFontBank::convert_game_to_utf8(const char* in) const {
   return replace_to_utf8(result);
 }
 
+/*!
+ * Same as convert_game_to_utf8, but gets rid of all special encodings like <COLOR_WHITE>.
+ */
+std::string GameTextFontBank::convert_game_to_utf8_special(const char* in) const {
+  std::string temp;
+  std::string result;
+  std::string test = in;
+  if (test.find("America") != std::string::npos) {
+    printf("");
+  }
```

Comment:
debugging leftover?

Reply:
I just copied `convert_game_to_utf8` verbatim and modified it a little bit, this wasn't from me, but I can get rid of it.